packages/dataviews/src/dataviews-layouts/grid/index.tsx

```diff
@@ -117,7 +118,7 @@ function GridItem< Item >( {
 				'is-selected': hasBulkAction && isSelected,
 			} ) }
 			onClickCapture={ ( event ) => {
-				if ( event.ctrlKey || event.metaKey ) {
+				if ( isAppleOS() ? event.metaKey : event.ctrlKey ) {
 					event.stopPropagation();
```

Comment:
Was this not working as expected? I can't tell the difference between trunk and this branch on the grid layout.

Reply:
As mentioned above, it wasn't working as expected on Windows - https://github.com/WordPress/gutenberg/pull/70891#discussion_r2230140291